tests/helpers.py

```diff
@@ -0,0 +1,23 @@
+"""
+Helper functions file for OCS QE
+"""
+import datetime
+
+
+def create_unique_object_name(object_description, object_type):
```

Comment:
I will change it to resource_description and resource_type

Reply:
+1
Also, I think we need to change the function name the same way as the args name. Something to..
"create_unique_resource_name"